MapboxNavigationUI/MGLMapView.swift

```diff
@@ -124,52 +136,42 @@ extension MGLMapView {
             
             maneuverArrowPolylines.append(headStrokePolyline)
             
-            let arrowSource = MGLShapeSource(identifier: "arrowSource", shape: MGLShapeCollection(shapes: maneuverArrowPolylines), options: nil)
-            let arrow = MGLLineStyleLayer(identifier: "arrow", source: arrowSource)
-            
-            arrow.lineWidth = MGLStyleValue(rawValue: 6)
-            arrow.lineColor = MGLStyleValue(rawValue: .white)
-            
-            // Arrow stroke
-            let arrowSourceStroke = MGLShapeSource(identifier: "arrowSourceStroke", shape: MGLShapeCollection(shapes: maneuverArrowStrokePolylines), options: nil)
-            let arrowStroke = MGLLineStyleLayer(identifier: "arrowStroke", source: arrowSourceStroke)
+            let arrowShape = MGLShapeCollection(shapes: maneuverArrowPolylines)
+            let arrowStrokeShape = MGLShapeCollection(shapes: maneuverArrowStrokePolylines)
             
             let cap = NSValue(mglLineCap: .round)
             let join = NSValue(mglLineJoin: .round)
             
-            arrowStroke.lineCap = MGLStyleValue(rawValue: cap)
-            arrowStroke.lineJoin = MGLStyleValue(rawValue: join)
-            arrow.lineCap = MGLStyleValue(rawValue: cap)
-            arrow.lineJoin = MGLStyleValue(rawValue: join)
+            let arrowSourceStroke = MGLShapeSource(identifier: arrowSourceStrokeIdentifier, shape: arrowStrokeShape, options: nil)
+            let arrowStroke = MGLLineStyleLayer(identifier: arrowSourceIdentifier, source: arrowSourceStroke)
+            let arrowSource = MGLShapeSource(identifier: arrowSourceIdentifier, shape: arrowShape, options: nil)
+            let arrow = MGLLineStyleLayer(identifier: arrowLayerIdentifier, source: arrowSource)
```

Comment:
arrow and arrowStroke should ideally be declared down in the else block instead of up here, since they aren't used in the if let block.